From 0610feb7639fe05e51debea59fc67fc09702d5da Mon Sep 17 00:00:00 2001
From: Rabin Vincent <rabin.vincent@axis.com>
Date: Thu, 28 May 2015 13:46:13 +0200
Subject: [PATCH] perf: fix MIPS build

5e8c0fb6a ("perf build: Add arch x86 objects building") added empty
Build files to several architectures, but MIPS got left out.  This leads
to a build of perf for MIPS failing with the following errors:

 tools/build/Makefile.build:40: arch/mips/Build: No such file or directory
 make[4]: *** No rule to make target `arch/mips/Build'.  Stop.

Add an empty Build file to fix this.

Signed-off-by: Rabin Vincent <rabin.vincent@axis.com>
[bva: taken from https://lkml.org/lkml/2015/5/28/242]
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 tools/perf/arch/mips/Build | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 tools/perf/arch/mips/Build

diff --git a/tools/perf/arch/mips/Build b/tools/perf/arch/mips/Build
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.1.0

